content/source/docs/enterprise/before-installing/reference-architecture/azure.md

```diff
@@ -96,6 +97,8 @@ in the Azure Blob Storage container. This allows for further
 encryption](https://docs.microsoft.com/en-us/azure/storage/common/storage-service-encryption)
 by Azure Blob Storage if required by your security policy.
 
+Due to support within the product, and increased durability, for a single-region deployment, we recommend using zone-redundant storage (ZRS) which synchronously writes across three Azure availability zones in the region.  For a multi-region deployment, use geo-zone-redundant storage (GZRS) for added region redundancy.
```

Comment:
What does "due to support within the product" mean?

Reply:
Clarified.